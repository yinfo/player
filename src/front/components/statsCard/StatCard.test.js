// @flow

import React from 'react';
import { shallow } from 'enzyme';
import StatsCard from './StatsCard';

describe('StatsCard component', () => {
  it('renders as expected', () => {
    const props = {
      statValue: '120',
      statLabel: 'label',
      icon: <i className="fa fa-check-square-o" />,
      backColor: 'blue',
    };

    const component = shallow(<StatsCard {...props} />);
    expect(component).toMatchSnapshot();
  });
});
