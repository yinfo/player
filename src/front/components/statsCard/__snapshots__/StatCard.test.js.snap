// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatsCard component renders as expected 1`] = `ShallowWrapper {}`;
